src/main/java/com/alkemy/ong/mapper/SlideMapper.java

```diff
@@ -43,4 +43,19 @@ public List<SlideDTOBasic> slideDTOListBasic(List<Slide> entities){
         return dtos;
     }
 
+    public List<SlideDTO> slideDTOList(List<Slide> entities){
+        List<SlideDTO> dtos = new ArrayList<>();
+        SlideDTO basicDTO;
+        for(Slide entity : entities){
```

Comment:
Ema, puede ser que dentro de este for puedas llamar el otro método para no repetir código?

Reply:
como seria?

Reply:
En este mismo mapper hay un método que se llama slide2DTO y hace lo mismo que está adentro de este for, o sea que en vez de setear uno a uno los atributos podrías llamar a ese método